audb/core/dependencies.py

```diff
@@ -259,17 +259,20 @@ def removed(self, file: str) -> bool:
         return self[file][define.DependField.REMOVED] != 0
 
     def load(self, path: str):
-        r"""Read dependencies from CSV file.
+        r"""Read dependencies from file.
 
         Clears existing dependencies.
 
         Args:
-            path: path to file
+            path: path to file.
+                File extension can be ``csv`` or ``pkl``.
 
         """
         self._df = pd.DataFrame(columns=define.DEPEND_FIELD_NAMES.values())
         path = audeer.safe_path(path)
-        if os.path.exists(path):
```

Comment:
I think we should raise an error if the path does not exist.

Reply:
Good idea, as this method is exposed to a user we should.

It now raises a `ValueError` if the file extension is not `csv` or `pkl` and raises a `FileNotFoundError` if the file does not exist.